src/stories/Library/opening-hours-editor/opening-hours-editor.scss

```diff
@@ -24,17 +24,28 @@
   padding: 0.5rem;
   color: #555;
   background-color: #fff;
+
+  &:disabled {
+    background-color: transparent;
+  }
 }
 
 .opening-hours-editor-form__checkbox {
-  margin-top: 0.75rem;
+  margin-top: 1.5rem;
   margin-bottom: 0.5rem;
   display: flex;
   gap: 0.5rem;
   align-items: center;
 
+  input[type="checkbox"] {
+    width: 25px;
+    height: 25px;
```

Comment:
Is there a reason why these dimensions are using `px` and not `rem` like elsewhere?